netbox/dcim/svg/cables.py

```diff
@@ -225,133 +235,93 @@ def draw_terminations(self, terminations):
             )
             nodes_height = max(nodes_height, node.box['height'])
             nodes.append(node)
+        return nodes, nodes_height
 
-        self.cursor += nodes_height
-        self.terminations.extend(nodes)
-
-        return nodes
+    def draw_terminations(self, terminations, parent_object_nodes):
+        """
+        Draw a row of terminating objects (e.g. interfaces) and return all created nodes
+        Attach them to previously created parent objects
+        """
+        nodes = []
+        nodes_height = 0
 
-    def draw_fanin(self, node, connector):
-        points = (
-            node.bottom_center,
-            (node.bottom_center[0], node.bottom_center[1] + FANOUT_LEG_HEIGHT),
-            connector.start,
-        )
-        self.connectors.extend((
-            Polyline(points=points, class_='cable-shadow'),
-            Polyline(points=points, style=f'stroke: #{connector.color}'),
-        ))
-
-    def draw_fanout(self, node, connector):
-        points = (
-            connector.end,
-            (node.top_center[0], node.top_center[1] - FANOUT_LEG_HEIGHT),
-            node.top_center,
-        )
-        self.connectors.extend((
-            Polyline(points=points, class_='cable-shadow'),
-            Polyline(points=points, style=f'stroke: #{connector.color}'),
-        ))
+        # Draw terminations for each parent object
+        for parent in parent_object_nodes:
+            parent_terms = [term for term in terminations if term.parent_object == parent.object]
 
-    def draw_cable(self, cable, terminations, cable_count=0):
-        """
-        Draw a single cable.  Terminations and cable count are passed for determining position and padding
+            if len(parent_terms) == 0:
+                self.logger.warn(f"No Parent Terminations? {parent.object.name}")
+                continue
```

Comment:
Did you actually come across this?  This feels like it should be a exception and not a logging of this.  I don't think there is any instance where an item should not have a parent.  Especially since you pull the parent object from the terminations anyways.